src/main/java/uk/gov/ch/repository/update/trusts/TrustDetailsRepository.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.ch.repository.update.trusts;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import uk.gov.ch.model.update.trusts.TrustDetails;
+
+import java.util.List;
+
+public interface TrustDetailsRepository extends PagingAndSortingRepository<TrustDetails, Long> {
+    @Query(value = "select trust_id, trust_name, creation_date, ceased_date, unable_to_obtain_all_info_ind "
```

Comment:
Alternatively:
"select t.trust_id, ca.corporate_body_appointment_id
from corporate_body cb
join corporate_body_appointment ca on ca.corporate_body_id = cb.corporate_body_id
join corbod_appt_trust_link cat on ca.corporate_body_appointment_id = cat.corporate_body_appointment_id
join trust t on t.trust_id = cat.trust_id
where cb.incorporation_number = ?"
[Added that to the ODP]